extensions/shellscript/package.json

```diff
@@ -67,6 +67,9 @@
           "bashrc_Apple_Terminal",
           "zshrc_Apple_Terminal"
         ],
+        "filenamePatterns": [
+          ".env*"
```

Comment:
Is the naming convention `.env*` or more like `.env.*`?

Reply:
@alexr00 you are right it make more sense to use `.env.*`. The rationale to do what I did was that I did the same for the Groovy language and `Jenkinsfile*` as seen here https://github.com/microsoft/vscode/pull/160385.

Granted though, that is much more specific in the naming in general, so maybe `.env.*` is better here. Let me know what you prefer.

Reply:
@thernstig I would prefer the more specific `.env.*` since it is less likely to match on files incorrectly.